docs/pages/guides/performance-management.md

```diff
@@ -0,0 +1,172 @@
+
+The dashboard allows us to run our Performance Tests , it also shows the different tests run over the lifetime of the profile sorted by time and day. 
+
+## Performance Management Through Meshery UI
+
+<a href="{{ site.baseurl }}/assets/img/performance-management/performance-run-test.PNG"><img alt="Performance Test Options" src="{{ site.baseurl }}/assets/img/performance-management/performance-run-test.PNG" /></a>
+
+Now that we have our setup, we can run a performance test with our required constraints. We can modify multiple testing scenarios by altering the Service Mesh used, adjusting the Concurrent Requests taking place or the number of queries run or select a particular Load Generator. For the sake of simplicity, we can also personalize the name of the test for future reference. 
+
+Upon personalization, select **Run Test** to execute the current test or **Save Profile** to save the properties of the test and execute it at a convinient time. 
+
+_SMI Conformance Test Results_
+
+<a href="{{ site.baseurl }}/assets/img/performance-management/perf-test-compare-option.PNG"><img alt="Performance Test Options" src="{{ site.baseurl }}/assets/img/performance-management/perf-test-compare-option.PNG" /></a>
+
+Once we run the tests and get the desired results, we can compare these test results against one another by selecting tests. For the best experience, you should compare between two or more tests of similar configuration. Tests that are configured with a high degree of variance (e.g., one test ran for a duration of 5 minutes, while another test ran for a duration of 1 hour) will produce comparisons that are more difficult to extrapolate insights from.
+
+Clicking on the **Compare Selected** button takes us to the next step of visualizing the comparision of the results.
+
+_Performance Test Results_
+
+<a href="{{ site.baseurl }}/assets/img/performance-management/performance-test-comparison.PNG"><img alt="Performance Test Results" src="{{ site.baseurl }}/assets/img/performance-management/performance-test-comparison.PNG" /></a>
+
+The results are depicted in the form of Histograms showing the count and Line Graphs showing the cumulative percentage.
+
+Latency and throughput are the two most significantly examined signals.
+
+Meshery will use different algorithms to calculate results depending upon which load generator was used to run the test.
+  
+_Viewing the Results_
+
+<a href="{{ site.baseurl }}/assets/img/performance-management/chart.png"><img alt="Performance Test Results" src="{{ site.baseurl }}/assets/img/performance-management/chart.png" /></a>
+
+## Performance Management Through Meshery CLI (mesheryctl)
+
+_Description_
+
+<!-- Description of the command. Preferably a paragraph -->
+{% assign name = site.data.mesheryctlcommands.cmds[page.command] %}
```

Comment:
Oh wait... I found why it didn't show up you need to include a command field on top of this file.... I'll show you...

Reply:
Had the same doubt! Thanks!